label_studio/users/urls.py

```diff
@@ -19,14 +19,18 @@
     path('user/signup/', views.user_signup, name='user-signup'),
     path('user/account/', views.user_account, name='user-account'),
     url(r'^logout/?$', views.logout, name='logout'),
-    # avatars
-    re_path(
-        r'^data/' + settings.AVATAR_PATH + '/(?P<path>.*)$',
-        serve,
-        kwargs={'document_root': join(settings.MEDIA_ROOT, settings.AVATAR_PATH)},
-    ),
     # Token
     path('api/current-user/reset-token/', api.UserResetTokenAPI.as_view(), name='current-user-reset-token'),
     path('api/current-user/token', api.UserGetTokenAPI.as_view(), name='current-user-token'),
     path('api/current-user/whoami', api.UserWhoAmIAPI.as_view(), name='current-user-whoami'),
 ]
+
+if not settings.CLOUD_FILE_STORAGE_ENABLED:
```

Comment:
Is the thinking here that production deployments always enable this setting, and therefore we can use `serve` in deployments that don't have cloud storage set up? I would suggest adding a comment explaining the rationale for gating this path behind this particular setting.

Reply:
When this setting is enabled, we upload avatars to the cloud storage and their url looks different, so we don't need `serve` and this path in that case.